test/functional/apps/home/_newsfeed.ts

```diff
@@ -11,12 +11,11 @@ import { FtrProviderContext } from '../../ftr_provider_context';
 
 export default function ({ getService, getPageObjects }: FtrProviderContext) {
   const globalNav = getService('globalNav');
-  const PageObjects = getPageObjects(['newsfeed']);
+  const PageObjects = getPageObjects(['newsfeed', 'common']);
 
-  // FLAKY: https://github.com/elastic/kibana/issues/135251
-  describe.skip('Newsfeed', () => {
+  describe('Newsfeed', () => {
     before(async () => {
-      await PageObjects.newsfeed.resetPage();
+      await PageObjects.common.navigateToUrl('home', undefined);
```

Comment:
NIT: Given you adapted the implementation of `PageObjects.newsfeed.resetPage`, I think you're good still using it?